validations/src/tests.rs

```diff
@@ -5434,13 +5435,14 @@ fn test_block_with_drpool<F: FnMut(&mut Block) -> bool>(
     mut_block: F,
     dr_pool: DataRequestPool,
 ) -> Result<(), failure::Error> {
-    test_block_with_drpool_and_utxo_set(mut_block, dr_pool, UnspentOutputsPool::default())
+    test_block_with_drpool_and_utxo_set(mut_block, dr_pool, UnspentOutputsPool::default(), 1000)
```

Comment:
The default should probably be after the FIRST_HARD_FORK, this way we ensure that all the existing tests still pass?

Reply:
But then we will have to change it again to SECOND_HARD_FORK in the future...